### PR TITLE
test: change file extension

### PR DESCRIPTION
**Pull request by talkor**
.jest => .test

### CODE REVIEW

File: packages/core/.eslintrc.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const commonExtends = ["plugin:react/recommended", "plugin:react-hooks/recommend
 module.exports = {
   overrides: [
     {
-      files: ["*.jest.js", "jest.init.js"],
+      files: ["*.test.js", "jest.init.js"],
```
Comment (YossiSaadi) [inline packages/core/.eslintrc.js:62]: it does not take into consideration `*.test.tsx` or `*.test.ts`
what is the meaning of it not catching it?

File: packages/core/plop/component/index.js [hunk 2]
```diff
@@ -53,13 +53,13 @@ module.exports = plop => {
...
-        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-snapshot-tests.jest.js",
-        templateFile: "plop/general/component-snapshot-tests-jest.txt"
+        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-snapshot-tests.test.js",
+        templateFile: "plop/general/component-snapshot-tests-test.txt"
       },
       {
         type: "add",
-        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-tests.jest.js",
-        templateFile: "plop/general/component-tests-jest.txt"
+        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-tests.test.js",
```
Comment (YossiSaadi) [inline packages/core/plop/component/index.js:61]: the file name dropped the `jest`, not replaced it with `test`

File: packages/core/plop/component/index.js [hunk 3]
```diff
@@ -53,13 +53,13 @@ module.exports = plop => {
       },
       {
         type: "add",
-        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-snapshot-tests.jest.js",
-        templateFile: "plop/general/component-snapshot-tests-jest.txt"
+        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-snapshot-tests.test.js",
```
Comment (YossiSaadi) [inline packages/core/plop/component/index.js:56]: the file name dropped the `jest`, not replaced it with `test`

File: packages/core/plop/general/component-snapshot-tests.txt
Comment (YossiSaadi) [inline packages/core/plop/general/component-snapshot-tests.txt:1]: you dropped the `jest` here

File: packages/core/plop/tests/index.js [hunk 4]
```diff
@@ -19,14 +19,13 @@ module.exports = plop => {
...
-          "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-snapshot-tests.jest.js",
-        templateFile: "plop/general/component-snapshot-tests-jest.txt"
+        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-snapshot-tests.test.js",
+        templateFile: "plop/general/component-snapshot-tests-test.txt"
       },
       {
         type: "add",
-        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-tests.jest.js",
-        templateFile: "plop/general/component-tests-jest.txt"
+        path: "src/components/{{properCase componentName}}/__tests__/{{camelCase componentName}}-tests.test.js",
```
Comment (YossiSaadi) [inline packages/core/plop/tests/index.js:27]: the file name dropped the `jest`, not replaced it with `test`

File: packages/storybook-blocks/.eslintrc.js [hunk 5]
```diff
@@ -39,7 +39,7 @@ const commonExtends = ['plugin:react/recommended', 'plugin:prettier/recommended'
 module.exports = {
   overrides: [
     {
-      files: ['*.jest.js', 'jest.init.js'],
+      files: ['*.test.js', 'jest.init.js'],
```
Comment (YossiSaadi) [inline packages/storybook-blocks/.eslintrc.js:42]: same as before, what's the meaning of it not catching ts{x}

File: packages/storybook-blocks/package.json [hunk 6]
```diff
@@ -40,7 +40,7 @@
     "prettier:fix": "prettier --write \"{,!(node_modules)/**/}*.{js,jsx,ts,tsx}\"",
     "test": "jest --passWithNoTests",
     "test:update": "yarn test -- -u",
-    "test:stories": "testing=storybook TEST_END_FILES=jest-story jest --passWithNoTests",
+    "test:stories": "testing=storybook TEST_END_FILES=test-story jest --passWithNoTests",
```
Comment (YossiSaadi) [inline packages/storybook-blocks/package.json:43]: I think it's redundant, we're not testing it, and also storyshots addon is deprecated
and anyway what the test did is:

```
**renderOnly**
Just render the story, don't check the output at all. This is useful as a low-effort way of smoke testing your components to ensure they do not error.
```

we already have this with Chromatic

Comment (YossiSaadi) [reply packages/storybook-blocks/package.json:43]: So you can remove `addon-storyshots` package as well + `stories.jest-story.ts` file

**Review by YossiSaadi — COMMENTED** [review]
Had a few question

File: packages/core/plop/general/component-snapshot-tests.txt
Comment (talkor) [reply packages/core/plop/general/component-snapshot-tests.txt:1]: I removed it completely, we don't use it anyway, we use snapshot testing inside the test files

File: packages/storybook-blocks/.eslintrc.js (hunk 5 above)
Comment (talkor) [reply packages/storybook-blocks/.eslintrc.js:42]: Removed it as it has no effect

**Pull request merged**